### PR TITLE
Ung 2218 update WebIdentId

### PR DESCRIPTION
**Pull request by wojciechUrbanski**
PR contains one additional change that is fixing typos in column names.

### CODE REVIEW

File: src/main/scala/com/ubirch/services/tenantadmin/TenantAdminService.scala [hunk 1]
```diff
@@ -62,6 +68,47 @@ class DefaultTenantAdminService @Inject() (pocRepository: PocRepository, pocAdmi
...
+      }
+    }
+
+    (for {
+      pocAdmin <- getPocAdmin(request.pocAdminId, UnknownPocAdmin)
+      _ <- isPocAdminAssignedToTenant[Task, UpdateWebIdentIdError](tenant, pocAdmin)(
+        PocAdminIsNotAssignedToRequestingTenant(pocAdmin.tenantId, tenant.id))
+      _ <- isSameWebIdentInitialId(tenant, pocAdmin)
+      pocAdminStatus <- getPocAdminStatus(pocAdmin)
+      _ <- updatePocAdminWebIdentId(pocAdmin)
```
Comment (Hiroki6) [inline src/main/scala/com/ubirch/services/tenantadmin/TenantAdminService.scala:107]: it's better to wrap here with transaction

File: src/main/scala/com/ubirch/db/tables/PocAdminTable.scala [hunk 2]
```diff
@@ -37,13 +41,25 @@ class PocAdminTable @Inject() (QuillMonixJdbcContext: QuillMonixJdbcContext) ext
...
 
+  private def updatePocAdminQuery(pocAdmin: PocAdmin) =
+    quote {
+      querySchema[PocAdmin]("poc_manager.poc_admin_table").filter(_.id == lift(pocAdmin.id)).update(lift(pocAdmin))
+    }
+
   private def updateWebInitiateId(pocAdminId: UUID, webInitiateId: UUID) =
     quote {
       querySchema[PocAdmin]("poc_manager.poc_admin_table").filter(_.id == lift(pocAdminId)).update(
         _.webIdentInitiateId -> lift(Option(webInitiateId))
```
Comment (Hiroki6) [inline src/main/scala/com/ubirch/db/tables/PocAdminTable.scala:52]: I missed here, it also should call `toString` ?

File: src/main/resources/db/migration/V6_0__poc_admin_status_typo_fixes.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+ALTER TABLE poc_manager.poc_admin_status_table
```
Comment (Hiroki6) [inline src/main/resources/db/migration/V6_0__poc_admin_status_typo_fixes.sql:1]: Sorry, please change the version number into V7 as V6 file is already in master

File: src/main/scala/com/ubirch/db/tables/PocAdminTable.scala (hunk 2 above)
Comment (Hiroki6) [reply src/main/scala/com/ubirch/db/tables/PocAdminTable.scala:52]: ahh, no it's fine.

File: src/main/scala/com/ubirch/services/tenantadmin/TenantAdminService.scala (hunk 1 above)
Comment (wojciechUrbanski) [reply src/main/scala/com/ubirch/services/tenantadmin/TenantAdminService.scala]: Indeed! :) Changed so it is run in transaction

File: src/main/resources/db/migration/V6_0__poc_admin_status_typo_fixes.sql (hunk 3 above)
Comment (wojciechUrbanski) [reply src/main/resources/db/migration/V6_0__poc_admin_status_typo_fixes.sql]: Done :)

File: src/main/scala/com/ubirch/db/tables/PocAdminTable.scala [hunk 4]
```diff
@@ -53,7 +70,16 @@ class PocAdminTable @Inject() (QuillMonixJdbcContext: QuillMonixJdbcContext) ext
   def getAllPocAdminsByTenantId(tenantId: TenantId): Task[List[PocAdmin]] = {
     run(getAllPocAdminsByTenantIdQuery(tenantId))
   }
+  override def updatePocAdmin(pocAdmin: PocAdmin): Task[Unit] = run(updatePocAdminQuery(pocAdmin)).void
+
   override def assignWebIdentInitiateId(pocAdminId: UUID, webIdentInitiateId: UUID): Task[Unit] = {
     run(updateWebInitiateId(pocAdminId, webIdentInitiateId)).void
   }
+  override def updateWebIdentIdAndStatus(webIdentId: UUID, pocAdminId: UUID): Task[Unit] =
```
Comment (Hiroki6) [inline src/main/scala/com/ubirch/db/tables/PocAdminTable.scala:78]: Just a remark, we can also use transaction in DefaultTenantAdminService, especially when we want to use transaction across many repositories. 
but this case is only two repositories, so I would say it's fine.

**Review by Hiroki6 — APPROVED** [review]

**Pull request merged**